cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -911,4 +912,17 @@ private MetadataScope validateScope(String scope) throws BadRequestException {
                                                   scope, MetadataScope.USER, MetadataScope.SYSTEM));
     }
   }
+
+  private EntityScope validateEntityScope(@Nullable String entityScope) throws BadRequestException {
+    if (entityScope == null) {
+      return EntityScope.ALL;
+    }
+
+    try {
+      return EntityScope.valueOf(entityScope.toUpperCase());
+    } catch (IllegalArgumentException e) {
+      throw new BadRequestException(String.format("Invalid entity scope '%s'. Expected '%s' , '%s' or null for both",
```

Comment:
what does it mean for user to pass in 'null' from the REST API?
They should just omit the parameter if they want both, right?